### PR TITLE
Web console guide: updating OCP version for Operator

### PR DESCRIPTION
**Pull request by RichardHoch**
Resolves: https://issues.redhat.com/browse/WINDUP-3008
Preview: http://file.emea.redhat.com/rhoch/16022021/ocpheadings/html-single/#installing_the_web_console_on_openshift_container_platform

### CODE REVIEW

**Comment by RichardHoch** [comment]
@apinnick: please review.

File: docs/topics_5/installing-web-console-on-openshift.adoc [hunk 1]
```diff
@@ -4,19 +4,19 @@
...
 ====
 The {ProductName} Operator is a Community Operator. Red Hat provides no support for Community Operators.
 ====
 endif::[]
 ifdef::ocp-41[]
-= Installing the {WebName} on {ocp-short} 4.2-4.4
+= Installing the {WebName} on {ocp-short} 4.2-4.5
 
-You can install the {WebName} on {ocp-short} 4.2-4.4 by importing a template and instantiating it to create the {WebName} application.
+You can install the {WebName} on {ocp-short} 4.2-4.5 by importing a template and instantiating it to create the {WebName} application.
```
Comment (apinnick) [inline docs/topics_5/installing-web-console-on-openshift.adoc:19]: This should be confirmed with QE. Did QE validate this doc change?

Comment (RichardHoch) [reply docs/topics_5/installing-web-console-on-openshift.adoc:19]: @apinnick -- yes. https://issues.redhat.com/browse/WINDUP-3008

**Comment by RichardHoch** [comment]
@apinnick: please review. QE has validated the change.

**Pull request merged**